collectors/osd.go

```diff
@@ -243,6 +247,20 @@ func NewOSDCollector(conn Conn, cluster string) *OSDCollector {
 			},
 			[]string{"osd"},
 		),
+
+		OSDExists: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace:   cephNamespace,
+				Name:        "osd_exists",
```

Comment:
Let's just name this `osd_tree`. That way it will be easier to understand what it means. We can also add more labels to it in future if need be.